flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/converters/SqlNodeConverter.java

```diff
@@ -37,6 +47,19 @@
      */
     Operation convertSqlNode(S node, ConvertContext context);
 
+    /**
+     * Returns the {@link SqlKind SqlKinds} of {@link SqlNode SqlNodes} that the {@link
+     * SqlNodeConverter} supports to convert.
+     *
+     * <p>If a {@link SqlNodeConverter} return s a non-empty SqlKinds, the conversion framework
+     * prefer to match SqlKind of SqlNode instead of matching class of SqlNode.
```

Comment:
Good catch. I should remove the "prefer". Here should be "The conversion framework will find the corresponding converter by matching the SqlKind of SqlNode instead of the class of SqlNode".